src/ocamlbuild_unix_plugin.ml

```diff
@@ -48,6 +48,14 @@ let at_exit_once callback =
   end
 
 let run_and_open s kont =
+  let s =
+    (* Be consistent! My_unix.run_and_open uses My_std.sys_command and
+       sys_command uses bash. *)
+    if Sys.win32 then
+      "bash --norc -c " ^ Filename.quote s
+    else
+      s
+  in
```

Comment:
Why is ` implem.run_and_open` overwritten in the first place?
I've assumed to avoid a temporary file, if `Unix.open_process_in` is used instead of `Sys.command`.